app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -66,6 +24,9 @@ def perform(_now)
       enrollments_in_progress: 0,
       enrollments_passed: 0,
     }
+    @request_delay_in_seconds = IdentityConfig.store.
```

Comment:
this looks fairly static, we could move to a constant? or do we override it in specs? (we can still use `stub_const` in specs if needed)

Reply:
I wasn't familiar with `stub_const` before reviewing this PR.  Thanks for sharing, Zach! 🙏🏻